mods/cnc/maps/gdi01/map.yaml

```diff
@@ -424,9 +424,6 @@ Rules:
 		-SpawnMPUnits:
 		-MPStartLocations:
 		-CrateSpawner:
-		PlayMusicOnMapLoad:
```

Comment:
But their map won't crash either. Maybe you could just remove the yaml node and print something along the lines of "`PlayMusicOnMapLoad` has been removed, use the Lua API's `PlayMusic` function instead" to the console